src/sudo.c

```diff
@@ -1169,10 +1169,8 @@ policy_open(void)
     if (ok != 1) {
 	if (ok == -2)
 	    usage();
-	else {
 	    /* XXX - audit */
```

Comment:
Please adjust the indentation of these two lines so it doesn't look like they belong with the `if (ok == -2)`